src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx

```diff
@@ -126,8 +130,57 @@ While we try our best to optimize how we display your data, sometimes you may ha
   <figcaption>
     **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Query your data**: This is an example of a query and how to select **Other groups** option.
   </figcaption>
-</Collapser>
 
+<Collapser
+    id="dash-color"
+    title="Color customization"
+>
+    You can customize the color of your charts per series in area, bar, line, pie, and stacked bar charts.
+  
+    To customize the color per series:
+    1. Run your query.
+    1. Open the Colors menu in the chart settings.
+    1. Select the plus <Icon name="fe-plus"/> icon.
+    1. Select the series you want to change the color.
+    1. Select the desired color.
```

Comment:
Same as above, should we change this to consecutive numbers?

Reply:
Yes, I'm going to change it. Thanks!